docs/reference/query-dsl/fuzzy-query.asciidoc

```diff
@@ -1,75 +1,99 @@
 [[query-dsl-fuzzy-query]]
 === Fuzzy Query
 
-The fuzzy query uses similarity based on Levenshtein edit distance.
+Returns documents that contain terms similar to the search term.
 
-==== String fields
+{es} measures similarity, or fuzziness, using a
```

Comment:
{es} measures similarity -> may be skip this sentence, or somehow indicate that  for this particular  query "similarity" (not a general notion of similarity in es).

Reply:
Removed with 9b97c9e.